### ISSUE TITLE
[Bug]: avatar should render an alt tag even if no label is specified to avoid a11y violations.

### ISSUE THREAD

**Issue by Rajdeepc** [adobe/spectrum-web-components#5513]
### Code of conduct

- [x] I agree to follow this project's code of conduct.

### Impacted component(s)

avatar

### Expected behavior

```html
<!-- After -->
<sp-avatar src="image.jpg"></sp-avatar>
<!-- Renders as: -->
<img class="image" alt="" src="image.jpg">
```

### Actual behavior

```html
<!-- Before -->
<sp-avatar src="image.jpg"></sp-avatar>
<!-- Rendered as: -->
<img class="image" src="image.jpg">
```

### What browsers are you seeing the problem in?

Microsoft Edge, Safari, Chrome, Firefox

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/avatar/#sizes
2. Remove label property
3. Open Dev Console
4. Check img doesnt have alt attribute

### Severity

None

### Would you like to track this issue in Jira?

- [x] Yes, please tell me the ticket number!

**Comment by Rajdeepc** [comment]
SWC-915

**Comment by jnurthen** [comment]
I don't think we should make avatars decorative if someone were to omit the `label`. I think that will hide too many real issues where someone just forgot to add a `label` to something which is not decorative.
If we are to do this I beleive we should use an intentional attribute a developer would specify (such as `isdecorative`). Until this time I think adding `aria-hidden="true"` to `sp-avatar` will solve the immediate issue.

**Comment by nikkimk** [comment]
When avatar is used as a link it absolutely must have a label. For accessibility anything focusable should be labeled and not `aria-hidden`
